app/src/main/java/com/example/pix/home/fragments/ChatsFragment.java

```diff
@@ -76,6 +78,26 @@ public void onLoadMore(int page, int totalItemsCount, RecyclerView view) {
             };
 
             rvChats.addOnScrollListener(scroll);
+
+            PullRefreshLayout layout = view.findViewById(R.id.swipeRefreshLayout);
+
+            // Whenever we try to refresh, delete all Chats, and get them again
+            layout.setRefreshStyle(PullRefreshLayout.STYLE_WATER_DROP);
+            layout.setOnRefreshListener(() -> {
+                try {
+                    Chat.getChatsInBackground(ParseUser.getCurrentUser(), 0, (objects, e) -> {
+                        chats.clear();
+                        chats.addAll(objects);
+                        chatsAdapter.notifyDataSetChanged();
+                        layout.setRefreshing(false);
+                    });
```

Comment:
Extra ideas for you to think about, not related to this PR.
Right now when you refresh, you download all the chats from the beginning of time and load them. Is it possible to only fetch the new chats since last time you fetched from the server?
Also, are the chats displayed in a certain order? For example, the chat with the newest message shows up on top.

Reply:
I can make it so it only fetches new Chats, but I think it may be better to not, because if a new Message was sent by a Chat that was at the bottom, I would want that Chat to be "refreshed" as being at the top.

In this branch, the Chats are not ordered, but I fixed this in a previous merge's update to the Chat.getChatsInBackground method.

Reply:
This might be a good opportunity for optimizations. When you think about "new chats", do you think of them in terms of the time it was created, or the last time it was updated? You can always fetch the chats that were updated since the last fetch.

Reply:
I'll keep this in mind in my newMessages branch, since that includes the latest version of the getChatsInBackground.